### ISSUE TITLE
Heapster UI is not accessible

### ISSUE THREAD

**Issue by rokka-n** [coreos/coreos-kubernetes#251]
```
vagrant up
...
==> default: Successfully added box 'coreos-alpha' (v926.0.0) for 'virtualbox'!

kubectl version
Client Version: version.Info{Major:"1", Minor:"1", GitVersion:"v1.1.2", GitCommit:"3085895b8a70a3d985e9320a098e74f545546171", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"1", GitVersion:"v1.1.2", GitCommit:"3085895b8a70a3d985e9320a098e74f545546171", GitTreeState:"clean"}

kubectl cluster-info
Kubernetes master is running at https://172.17.4.99:443
Heapster is running at https://172.17.4.99:443/api/v1/proxy/namespaces/kube-system/services/heapster
KubeDNS is running at https://172.17.4.99:443/api/v1/proxy/namespaces/kube-system/services/kube-dns
KubeUI is running at https://172.17.4.99:443/api/v1/proxy/namespaces/kube-system/services/kube-ui
InfluxDB is running at https://172.17.4.99:443/api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb

kubectl proxy
Starting to serve on 127.0.0.1:8001
```
Open in browser http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/heapster
Error: 'dial tcp 10.2.76.5:8082: connection refused'
Trying to reach: 'http://10.2.76.5:8082/'

Also, url for influxdb should include ":http"
http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb:http

**Comment by joeatwork** [comment]
Thanks for the report!

That error is coming from Heapster itself, so it looks like you've hit the UI, such as it is (there really isn't much to the heapster UI) - that error presents itself when the influxdb pod isn't available. Often if you'll wait a bit (for me, "a bit" in a single-node vagrant install might even be a few minutes) you'll see the heapster UI. You can check to see if both the pods are up and running with something like
```sh
$ kubectl --namespace=kube-system get pods
NAME                                  READY     STATUS    RESTARTS   AGE
heapster-v10-35bva                    1/1       Running   0          14m
kube-apiserver-172.17.4.99            1/1       Running   0          14m
kube-controller-manager-172.17.4.99   1/1       Running   0          14m
kube-dns-v9-wklyn                     4/4       Running   0          13m
kube-proxy-172.17.4.99                1/1       Running   0          14m
kube-scheduler-172.17.4.99            1/1       Running   0          14m
monitoring-influxdb-v2-b8n2b          1/1       Running   0          14m
```

Be aware, though, that even after the monitoring-influxdb pod comes up, you'll still need to wait a bit - Heapster only tries to write about once per minute.

Also, once things come up, be aware that the default Heapster UI isn't really very `kubernetes proxy` savvy, and you'll likely have to hit a URL like

    http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/heapster/validate/

To see it. Please comment here if this (waiting for the pods to stabilize) addresses your issue.

We don't expect many folks to be accessing the Heapster or InfluxDB UI's directly - they're provided in the image for the most part to allow for horizontal scaling. We'd love to hear about what you're using these services for!

**Comment by rokka-n** [comment]
Thank you for suggestions! 
Not sure if it was a problem with one of previous versions, delayed vagrant provisioning but the latest one works fine.  There is a correct status page at validate url

I wanted to get heapster working to check kubedash panel
```
git clone https://github.com/kubernetes/kubedash.git; cd kubedash
kubectl create -f deploy/kube-config.yaml
kubectl proxy
```
The site becomes available at http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/kubedash/#!/

**Comment by joeatwork** [comment]
Thanks for following up!
